server/api/status_sync_job.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package api
+
+import (
+	"sync"
+	"time"
+
+	"github.com/mattermost/mattermost-plugin-mscalendar/server/utils/bot"
+)
+
+const JOB_INTERVAL = 5 * time.Minute
+
+type StatusSyncJob struct {
+	api        API
```

Comment:
This is confusing because we usually call the plugin API this.

Reply:
I think that once the other is renamed to `PluginAPI` and `p` is not replicated in the code, `api` becomes self-explanatory; but I'd love a better name; maybe `app` (for everything, package and all) despite all of its negative properties?

Reply:
Separate ticket for renaming package if we plan on doing so?